alstof-jeremy/README.md

```diff
@@ -13,10 +13,18 @@ Clone the repo. Open index.html in your browser.
 
 ## Architecture
 
-The app is built using HTML, CSS, Javascript, JQuery library and SMACSS methodolody.
+The app is built using:
+
+- HTML,
+- CSS,
+- Javascript,
+- JQuery library,
+- SMACSS methodolody.
 
 ## Change Log
-09-06-2018 11:03am - Implement read on functionality to display full article. 
+
```

Comment:
make sure you are previewing the markdown file before submitting. this section needs to be made a list too so it's not all jumbled on one line